cron/poll_issue_close_stale.py

```diff
@@ -0,0 +1,45 @@
+import arrow
+import logging
+
+import settings
+import github_api as gh
+
+__log = logging.getLogger("poll_close_stale")
+
+
+def poll_issue_close_stale():
+    """
+    Looks through all open issues. For any open issue, if the issue is
+    too old and has not been recently commented on, chaosbot issues a
+    /vote close...
+    """
+
+    __log.info("Checking for stale issues...")
+
+    api = gh.API(settings.GITHUB_USER, settings.GITHUB_SECRET)
```

Comment:
Ah, I was wondering if there was a catch to using `gh.API` everywhere... Will update and push